<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imgSegmentation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="myLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox{border:2px solid #a1a1a1; 
border-radius:10px;}</string>
      </property>
      <property name="title">
       <string>srttings</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; 
width:0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Bilateral  parameter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; 
width:0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Spatial grid size</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid #a1a1a1;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>50</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid #a1a1a1;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>16</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; 
width:0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Color grid size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; 
width:0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Graph cut parameter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; 
width:0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>E2 Weight </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>240</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid #a1a1a1;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>10.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>change</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open"/>
   <addaction name="action"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="run"/>
   <addaction name="save"/>
   <addaction name="about"/>
  </widget>
  <action name="open">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_24.png</normaloff>res/simple_vvr_24.png</iconset>
   </property>
   <property name="text">
    <string>打开图像</string>
   </property>
   <property name="toolTip">
    <string>open image</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_01.png</normaloff>res/simple_vvr_01.png</iconset>
   </property>
   <property name="text">
    <string>保存分割结果</string>
   </property>
   <property name="toolTip">
    <string>save segmentation result</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_48.png</normaloff>res/simple_vvr_48.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>about</string>
   </property>
  </action>
  <action name="run">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_51.png</normaloff>res/simple_vvr_51.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>run</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_22.png</normaloff>res/simple_vvr_22.png</iconset>
   </property>
   <property name="text">
    <string>标记背景</string>
   </property>
   <property name="toolTip">
    <string>mark background</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_19.png</normaloff>res/simple_vvr_19.png</iconset>
   </property>
   <property name="text">
    <string>标记前景</string>
   </property>
   <property name="toolTip">
    <string>mark foreground</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normaloff>res/simple_vvr_07.png</normaloff>res/simple_vvr_07.png</iconset>
   </property>
   <property name="text">
    <string>重新标记</string>
   </property>
   <property name="toolTip">
    <string>remark</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
